recipes/snompy/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "snompy" %}
+{% set version = "0.1.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/snompy-{{ version }}.tar.gz
+  sha256: 9e344f28b89bf5f75ecffd3bcf5fc3ccad0832e4ffc0f3bdffb18e6f9b7b21c0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - pip
+  run:
+    - python >=3.10
+    - numpy
+
+test:
+  imports:
+    - snompy
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/TomVincentUK/snompy
+  summary: A Python package for modelling scanning near-field optical microscopy measurements.
+  license: MIT
+  license_file: LICENCE.txt
+
+extra:
+  recipe-maintainers:
+    - TomVincentUK # I am happy to be maintainer - TomVincentUK
```

Comment:
No need to put this inline comment. Otherwise the recipe looks good

Reply:
Thanks very much! I've removed the unneeded comment. Is it ready to merge?